### ISSUE TITLE
rtx-bin sha512 not being updated

### ISSUE THREAD

**Issue by inverse** [jdx/rtx#1148]
**Describe the bug**

Something changed with `2023.12.22` that the sha512sum stopped getting updated.

See: https://aur.archlinux.org/cgit/aur.git/commit/?h=rtx-bin&id=4616ad17e73c43d1e8819e3ec1c8c4fe41f183c4

It was working before on `2023.12.21`.

**To Reproduce**

Using your aur helper of choice install `rtx-bin`. e.g. with `yay` this will yield.

```
==> Validating source files with sha512sums...
    rtx-2023.12.24.tar.gz ... FAILED
==> ERROR: One or more files did not pass the validity check!
 -> error downloading sources: /home/<redacted>/.cache/yay/rtx-bin 
	 context: exit status 1 
```

**Expected behavior**

`rtx-bin` to install properly.

**`rtx doctor` output**

N/A

**Additional context**

N/A

**Comment by jdx** [comment]
I'm not really sure what is happening but I think it might be because it's getting an HTTP error. If that's the case, https://github.com/jdx/rtx/commit/b88674161a574cc1865ada7743a96b9d3e927330 will at least prevent it from publishing.

**Comment by jdx** [comment]
I pushed an update for at least the latest revision to have the correct sha

**Comment by inverse** [comment]
Yeah it seems really odd. Hopefully that'll uncover the issue.

**Comment by inverse** [comment]
@jdx will close this as can confirm fixed `2023.12.25`.

Thanks for a quick turn around :pray:
